src/systemctl/systemctl-logind.c

```diff
@@ -45,15 +45,16 @@ int logind_reboot(enum action a) {
 #if ENABLE_LOGIND
         static const struct {
                 const char *method;
+                const char *method_new;
```

Comment:
Please don't call this "_new", but rarther "_with_flags" or so. (Those things tend to age quickly, and then "new" is "old" and "new new" is "not so old" and "the newest new" is the new "new".)